### PR TITLE
virtman_view: stabilize items selection in graph menu of virt manager (poo#54056)

### PR DESCRIPTION
**Pull request by ggardet**
- Related ticket: https://progress.opensuse.org/issues/54056
- Verification run: 
  * x86_64: https://openqa.opensuse.org/t1078600
  * aarch64: https://openqa.opensuse.org/t1078601

### CODE REVIEW

File: tests/virtualization/virtman_view.pm [hunk 1]
```diff
@@ -50,8 +50,13 @@ sub run {
...
     # go to view now
-    wait_screen_change { send_key 'alt-v' };
-    wait_screen_change { send_key 'right' };
+    # decompose alt-v - poo#54056
+    hold_key 'alt';
+    send_key 'v';
+    sleep(1);
+    release_key 'alt';
+    send_key 'right';
+    assert_screen 'virtman-viewmenu-graph';
```
Comment (rwx788) [inline tests/virtualization/virtman_view.pm]: From the code I can see that this module is used for SLES too, so we might need to create needle for it too. Let me check.

Comment (rwx788) [reply tests/virtualization/virtman_view.pm]: I've triggered https://openqa.suse.de/tests/3568360 (paste it here not to loose the link), will use it to create the needle as you did for TW and then we can merge this PR.

**Comment by rwx788** [comment]
> I think the commit message is incomplete. I would say the important part here is not the assert_screen, but the way we make the machine be slow like a human.
 It's still `assert_screen` to assure that we gt the dialog before continuing hitting the keys. But I agree commit message should reflect purpose, like `stabilize items selection in graph menu of virt manager`.
@ggardet WDYT?

**Comment by ggardet** [comment]
> @ggardet WDYT?

Ok, I updated the commit message and the PR title.

**Comment by coolo** [comment]
Can one of you please file a ticket to get a slow_send_key implementing this in testapi.pm? Or do you find it reasonable to always do that?

**Comment by coolo** [comment]
Stole ￼￼https://progress.opensuse.org/issues/55703

**Comment by ggardet** [comment]
> Can one of you please file a ticket to get a slow_send_key implementing this in testapi.pm? Or do you find it reasonable to always do that?

I would suggest to merge this one as is for now to fix the problem and I start to work on `slow_send_key` implementation.

**Comment by SergioAtSUSE** [comment]
@ggardet, I would merge, waiting for @rwx788's feedback

**Comment by ggardet** [comment]
https://github.com/os-autoinst/os-autoinst/pull/1259 fixes it properly with a `send_key` update. 
So, we can skip this PR if https://github.com/os-autoinst/os-autoinst/pull/1259 is merged and deployed soon.

**Pull request merged**